Automattic/jetpack
# Contact Form: Remove the Connection button from the Form block toolbar

Fixes #27294

#### Changes proposed in this Pull Request:
* Remove the Connection button from the Form block toolbar
* More context on #27294

Before:
<img width="333" alt="image" src="https://user-images.githubusercontent.com/7811225/200426171-228628f8-c712-4bd6-824b-e8c7625a3556.png">

After
<img width="154" alt="image" src="https://user-images.githubusercontent.com/7811225/200426348-e65732b9-57f1-4e1e-9c41-baa1a60de1ab.png">


#### Other information:


#### Jetpack product discussion
N/A

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
* Create a Post/Page and add a Form block
* Check if the Connection button was removed from the From block toolbar
